src/components/developerCards/index.jsx

```diff
@@ -71,67 +70,61 @@ class DeveloperCard extends Component {
   );
 
   renderCards = () => {
-    const { data, openModal, changeModalTypes } = this.props;
+    const { card, openModal, changeModalTypes } = this.props;
     const automation = length => length !== 0;
     const automationMetric = (activities, status) => (
       _.filter(activities, { status }).length
     );
     const metric = (activities, status) => (
       automation(activities.length) ? automationMetric(activities, status) : 0
     );
-    return data.map((card, index) => {
-      const name = card.fellowName;
-      const newName = _.split(name, ',');
-      const firstInitials = newName[0].charAt(0);
-      const secondInitials = newName[1] && newName[1].charAt(1);
-      return (
-        <div className="card" key={card.id}>
-          <div className="info-cont">
-            {card.type === 'onboarding'
-              ? (
-                <div className="tooltip-container" id="onboarding-info-icon">
-                  <img className="info-icon onBoarding-icon" src={onboarding} alt="onboarding icon" />
-                  <span className="tooltiptext">On-boarding</span>
-                </div>
-              ) : (
-                <div className="tooltip-container" id="offboarding-info-icon">
-                  <img className="info-icon onBoarding-icon" src={offboarding} alt="offboarding icon" />
-                  <span className="tooltiptext">Off-boarding</span>
-                </div>
-              )}
-            <div id="more-info-icon" className="tooltip-container" onClick={() => { openModal(); changeModalTypes(card); }}>
-              <img className="info-icon" src={InfoIcon} alt="info icon" role="presentation" id={`${index}-id`} />
-              <span className="tooltiptext">Details</span>
-            </div>
-          </div>
-          {this.renderDeveloperPic(firstInitials, secondInitials)}
-          <div className="clickableCardContent tooltip-container" id="fellow-name" onClick={() => window.open(`https://ais.andela.com/people/${card.fellowId}`)}>
-            {this.renderDetails('developerDetails', 'developerName', card.fellowName)}
-            <span className="tooltip-icon">
-              <i className="fas fa-external-link-alt" />
-            </span>
+    const name = card.fellowName;
+    const newName = _.split(name, ',');
+    const firstInitials = newName[0].charAt(0);
+    const secondInitials = newName[1].charAt(1);
```

Comment:
This change is likely to return the bug that was solved [here](https://www.pivotaltracker.com/story/show/167191688) and fixed through this [PR](https://github.com/andela/bp-esa-frontend/pull/101)

Reply:
@ssewilliam, the application is not returning the bug you have referenced in the `story` and `PR` above